src/core/middlewares/url/handler.go

```diff
@@ -37,7 +37,7 @@ func New(next http.Handler) http.Handler {
 
 // ServeHTTP ...
 func (uh urlHandler) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
-	log.Debugf("in url handler, path: %s", req.URL.Path)
+	log.Infof("in url handler, path: %s", req.URL.Path)
```

Comment:
Is this log useful to customers? If not, suggest to keep it same before as debug message.

Reply:
resolved, thanks.